<?xml version="1.0" encoding="utf-8"?>
<launch>
  <arg name="erosion_size"                 default="2"/>

  <!-- Elevation map filter node. -->
  <node pkg="grid_map_filters_drs" type="grid_map_erosion_node" name="grid_map_erosion_node" output="screen" >
    <param name="erosion_size" value="$(arg erosion_size)" />
  </node>

</launch>
